specifyweb/stored_queries/tests.py

```diff
@@ -1357,5 +1357,23 @@ def test_sqlalchemy_model_errors(self):
         "onetoone"
       ]
     }
-  }
+  },
+  "CollectionObjectGroupJoin": {
+    "incorrect_direction": {
+      "childcog": [
+        "manytoone",
+        "onetoone"
+      ],
+      "childco": [
+        "manytoone",
+        "onetoone"
+      ]
+    }
+  },
+  "CollectionObjectGroup": {
+    "not_found": [
+      "cojo",
+      "parentcojos"
```

Comment:
Be careful here. Not seeing these actually means you won't be able to query them. That is, if you were do a COG -> cojo/parentcojos, it'll throw an error. Basically, every field should always be here, epeect m2m's

Reply:
Actually, just need to define othersidename. that should be all.

Reply:
@acwhite211, I saw the other CO failing (projects one). You need to define the fields correctly (the otherSideName). You shouldn't need to touch the tests file's not found at all in any case, modify rels to achieve that.